libraries/Microsoft.Bot.Builder.AI.LUIS/Microsoft.Bot.Builder.AI.LUIS.csproj

```diff
@@ -62,10 +62,10 @@
 	</PropertyGroup>
 
```

Comment:
The 3 CodeAnalysisRuleSet elements above, in the conditional Elements, should be removed. The ruleset shouldn't be conditional. A non-conditional once is already present on line 59 which is perfect.

Reply:
Done